client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -9,7 +9,7 @@
     </div>
 
     <div class="button-stack right-stack">
-        <button class="btn btn-medium pull-left" ng-if="item._locked && can_unlock() && itemActions.save" ng-click="unlock()" ng-disabled="unlockClicked" translate>Unlock</button>
+        <button class="btn btn-medium pull-left" ng-if="item._locked && !item.more_coming && can_unlock() && itemActions.save" ng-click="unlock()" ng-disabled="unlockClicked" translate>Unlock</button>
```

Comment:
I suggest wrapping  this line as it is almost 200 characters long, requiring substantial horizontal scrolling.
Maybe also consider putting the `ng-if` condition logic into the controller, it's starting to become a bit too complex to have it in the template.